experiments/webext/src/utils/settings.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import browser from 'webextension-polyfill'
+
+interface Settings {
+  host: string | null
+  username: string | null
+  token: string | null
+  boardId: string | null
+}
+ 
+export function loadSettings(): Settings {
+  return browser.storage.sync.get(['host', 'username', 'token', 'boardId'])
```

Comment:
Do you know if the namespace for localstorage is per extension? i.e. could these property names collide with other code?